_posts/2019-07-29-SageDays102.md

```diff
@@ -0,0 +1,434 @@
+---
+layout: post
+author: Erik Madison Bray
+title: Report on Sage Days 102, University of Ibadan, Nigeria
+image: SageDays102/group.jpg
+tags:
+    - blogpost
+    - workshop
+    - dissemination
+    - sagemath
+    - front-page
+---
+
+During the week of July 15 - 19 we held [Sage Days
+102](https://wiki.sagemath.org/days102) at the University of Ibadan, in
+Ibadan, Oyo State, Nigeria. This was (to our knowledge) the first ever
+week-long workshop dedicated primarily to SageMath in Nigeria (although not
+the first ever in West Africa; that honor belongs to [Sage Days
+34.5](https://wiki.sagemath.org/days34.5) in Burkina Faso, October 2011),
+and there was immense interest, with 80 registered participants, and many
+others who had to be wait-listed.
+
+The primary focus of the workshop was to introduce students and faculty to
+computational mathematics with SageMath, as well as to on-board them to open
+source mathematical and scientific computing in general. To that end we also
+held sessions on GAP and R with the help of local experts.
+
+All of the on-site organization and funding was provided by the University
+of Ibadan Department of Mathematics, while OpenDreamKit provided travel
+funding for four instructors (Erik Madison Bray from France, Yaé Ulrich Gaba
+from South Africa, Evans Doe Ocansey from Austria, and Dr. Chimere Stanley
+Anabanti from Nigeria), as well as for two PhD students (Olivia Adu-Poku
+from Ghana and Gilda Bansimba from Congo). The instructors were also joined
+by Dr. Olalekan K. Obisesan, a professor of statistics local to the
+University of Ibadan.
+
+The genesis of this workshop was at the [Free Computational
+Mathematics](https://opendreamkit.org/2019/02/11/FreeComputationalMathematicsConference/)
+conference, held at CIRM in France, February 2019, also sponsored by
+OpenDreamKit.  That conference was attended by Ini Adinya, a lecturer in
+mathematical finance at the University of Ibadan, along with two of her
+colleagues.  They were impressed by how much they were learning at that
+conference, and wanted to replicate it, or something like it, at the
+University of Ibadan.  Ini and Erik had worked together during the
+conference on some of her programming problems, and Erik agreed to come help
+out and teach at such a workshop if they organized it.  And so, shortly
+afterwards we got the ball rolling, with a big kick of help from Viviane
+Pons who helped with much of the initial organizing, and especially the
+search for additional instructors who would be willing to travel to (or from
+within) Nigeria, as without several instructors there was no way the
+workshop could be successful.  One of the IT managers at AIMS, the African
+Institute for Mathematical Sciences, saw Viviane's e-mail on one of the Sage
+mailing lists and forwarded it to Evans and Yaé, who had worked together
+before teaching Sage in South Africa.  They in turn agreed it would be fun
```

Comment:
Instead of "who had worked together before teaching Sage in South Africa", maybe "who had taught Sage together before in South Africa"